packages/vscode/src/plugins/prisma-language-server/index.ts

```diff
@@ -59,41 +56,21 @@ const activateClient = (
   context.subscriptions.push(disposable)
 }
 
-const startFileWatcher = (rootPath: string) => {
-  console.debug('Starting File Watcher')
-  // https://github.com/fabiospampinato/watcher
-  return new FileWatcher(rootPath, {
-    debounce: 500,
-    // limits how many levels of subdirectories will be traversed.
-    // Note that `node_modules/.prisma/client/` counts for 3 already
-    // Example
-    // If vs code extension is open in root folder of a project and the path to index.d.ts is
-    // ./server/database/node_modules/.prisma/client/index.d.ts
-    // then the depth is equal to 2 + 3 = 5
-    depth: 9,
-    recursive: true,
-    ignoreInitial: true,
-    ignore: (targetPath) => {
-      if (targetPath === rootPath) {
-        return false
-      }
-      return !minimatch(targetPath, '**/node_modules/.prisma/client/index.d.ts')
-    },
-    //   // ignore dotfiles (except .prisma) adjusted from chokidar README example
-    //   ignored: /(^|[\/\\])\.(?!prisma)./,
-
-    // native?: boolean,
-    // persistent?: boolean,
-    // pollingInterval?: number,
-    // pollingTimeout?: number,
-    // renameDetection?: boolean,
-    // renameTimeout?: number
-  })
+const onFileChange = (filepath: string) => {
+  console.debug(`File ${filepath} has changed, restarting TS Server.`)
+  void commands.executeCommand('typescript.restartTsServer')
 }
 
-const onFileChange = (filepath: string) => {
-  console.debug(`File ${filepath} has been changed. Restarting TS Server.`)
-  commands.executeCommand('typescript.restartTsServer') // eslint-disable-line
+const startGenerateWatcher = () => {
+  const prismaCache = paths('checkpoint').cache
+  const signalsPath = path.join(prismaCache, 'last-generate')
+  const fwOptions = { debounce: 500, ignoreInitial: true }
+  const fw = new FileWatcher(signalsPath, fwOptions)
```

Comment:
We now watch a single file.

Reply:
Can you clarify what you mean here by other generators?